client/a8c-for-agencies/sections/team/primary/team-invite/index.tsx

```diff
@@ -48,20 +60,38 @@ export default function TeamInvite() {
 							duration: 5000,
 						} )
 					);
+					dispatch( recordTracksEvent( 'calypso_a4a_team_invite_success' ) );
 					page( `${ A4A_TEAM_LINK }/${ TAB_INVITED_MEMBERS }` );
 				},
 
 				onError: ( error ) => {
 					dispatch( errorNotice( error.message ) );
+					dispatch(
+						recordTracksEvent( 'calypso_a4a_team_invite_error', {
+							error: 'api_error',
+						} )
+					);
 				},
 			}
 		);
 	}, [ dispatch, message, sendInvite, translate, username ] );
 
-	const onUsernameChange = useCallback( ( value: string ) => {
-		setError( '' );
-		setUsername( value );
-	}, [] );
+	const onUsernameChange = useCallback(
+		( value: string ) => {
+			setError( '' );
+			setUsername( value );
+			dispatch( recordTracksEvent( 'calypso_a4a_team_invite_username_change' ) );
```

Comment:
Same as above. I don't think we need to track these events. What do you think?

Reply:
Now that you've mentioned it, this might end up being a spammy event. Checking on how we did it before; we actually just tracked the onClick event. I'll update this one. Thanks for bringing this up! 👍